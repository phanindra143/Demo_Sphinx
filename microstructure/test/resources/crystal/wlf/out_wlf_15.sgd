*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=   107.1509
  !k_ij(1,2)=     1.1109
  !k_ij(2,2)=   106.8898
  !k_ij(1,3)=    -0.7658
  !k_ij(2,3)=     0.7285
  !k_ij(3,3)=   106.2031
MP,Kxx,1,  107.15093
MP,Kyy,1,  106.88985
MP,Kzz,1,  106.20313
MP,Kxx,2,  106.88985
MP,Kyy,2,  107.15093
MP,Kzz,2,  106.20313
MP,Kxx,3,  106.88985
MP,Kyy,3,  107.15093
MP,Kzz,3,  106.20313
MP,Kxx,4,  107.15093
MP,Kyy,4,  106.88985
MP,Kzz,4,  106.20313
MP,Kxx,5,  107.15093
MP,Kyy,5,  106.88985
MP,Kzz,5,  106.20313
MP,Kxx,6,  106.88985
MP,Kyy,6,  107.15093
MP,Kzz,6,  106.20313
MP,Kxx,7,  106.88985
MP,Kyy,7,  107.15093
MP,Kzz,7,  106.20313
MP,Kxx,8,  107.15093
MP,Kyy,8,  106.88985
MP,Kzz,8,  106.20313
MP,Kxx,9,  106.20313
MP,Kyy,9,  107.15093
MP,Kzz,9,  106.88985
MP,Kxx,10,  107.15093
MP,Kyy,10,  106.20313
MP,Kzz,10,  106.88985
MP,Kxx,11,  107.15093
MP,Kyy,11,  106.20313
MP,Kzz,11,  106.88985
MP,Kxx,12,  106.20313
MP,Kyy,12,  107.15093
MP,Kzz,12,  106.88985
MP,Kxx,13,  106.20313
MP,Kyy,13,  107.15093
MP,Kzz,13,  106.88985
MP,Kxx,14,  107.15093
MP,Kyy,14,  106.20313
MP,Kzz,14,  106.88985
MP,Kxx,15,  107.15093
MP,Kyy,15,  106.20313
MP,Kzz,15,  106.88985
MP,Kxx,16,  106.20313
MP,Kyy,16,  107.15093
MP,Kzz,16,  106.88985
MP,Kxx,17,  106.88985
MP,Kyy,17,  106.20313
MP,Kzz,17,  107.15093
MP,Kxx,18,  106.20313
MP,Kyy,18,  106.88985
MP,Kzz,18,  107.15093
MP,Kxx,19,  106.20313
MP,Kyy,19,  106.88985
MP,Kzz,19,  107.15093
MP,Kxx,20,  106.88985
MP,Kyy,20,  106.20313
MP,Kzz,20,  107.15093
MP,Kxx,21,  106.88985
MP,Kyy,21,  106.20313
MP,Kzz,21,  107.15093
MP,Kxx,22,  106.20313
MP,Kyy,22,  106.88985
MP,Kzz,22,  107.15093
MP,Kxx,23,  106.20313
MP,Kyy,23,  106.88985
MP,Kzz,23,  107.15093
MP,Kxx,24,  106.88985
MP,Kyy,24,  106.20313
MP,Kzz,24,  107.15093
*endif
