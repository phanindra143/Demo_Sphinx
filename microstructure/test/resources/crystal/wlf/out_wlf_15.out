            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\WLF-50SiC-Poren\7
             3.00000      -1674198.78537        -22401.28786          2378.94153
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000            46.34379           100.78221
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000        -19444.60475      -1670029.58236         25511.67482
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000            17.37575         15625.00000          -314.33513
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000         -1438.10557         25742.16969      -1658992.64744
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000           118.26733          -381.30864         15625.00000
  /%
